front/src/modules/ui/table/table-header/components/TableHeader.tsx

```diff
@@ -72,12 +72,15 @@ export function TableHeader<SortField>({
           <FilterDropdownButton
             context={TableContext}
             HotkeyScope={FiltersHotkeyScope.FilterDropdownButton}
+            isPrimaryButton
```

Comment:
I'm not sure to understand what `isPrimaryButton` means?

Reply:
There are two filter buttons available: one located at the top and the other within the sort and filter bar. The top button possesses distinct attributes. For instance, its behavior upon clicking varies based on whether a filter already exists or not. The button in the sort-and-filter bar is a reused version of the top button. However, its behavior remains consistent, simply adding a filter without considering the presence of an existing one. To differentiate between these two buttons, we refer to the main button as `isPrimaryButton`.

Reply:
I think it's ok for now but it's not ideal.